### ISSUE TITLE
oneplus.com - Assistant icon not visible

### ISSUE THREAD

**Issue by danidoedel** [AdguardTeam/AdguardAssistant#281]
https://www.oneplus.com/de

![image](https://user-images.githubusercontent.com/10995998/54883273-806f8880-4e64-11e9-9ee2-da09f9bcc1fd.png)

**Comment by danidoedel** [comment]
@ameshkov Ok seems to be a more general problem as it is not working on many sites. I am using AdGuard for Windows 7.0.2372.6019. Example console log:

[www.heise.de-1553449757264.txt](https://github.com/AdguardTeam/AdguardAssistant/files/3000642/www.heise.de-1553449757264.txt)

**Comment by ameshkov** [comment]
```
TypeError: a.m is not a function
    at eval (Adguard Assistant Beta.user.js:3)
    at eval (Adguard Assistant Beta.user.js:3)
    at settings.scriptInfo.script.options.run-at (?ts=1553448871505&name=Adguard Assistant Beta&name=AdGuard Popup Blocker (Beta)&type=user-script:42)
    at ?ts=1553448871505&name=Adguard Assistant Beta&name=AdGuard Popup Blocker (Beta)&type=user-script:40
    at Array.forEach (<anonymous>)
    at settings.scriptInfo.script.options.run-at (?ts=1553448871505&name=Adguard Assistant 
```

**Comment by ameshkov** [comment]
What's strange is that it works in my case. Could you please try reinstalling AG Assistant just in case?

**Comment by danidoedel** [comment]
@ameshkov No, that did not help :(

**Comment by danidoedel** [comment]
Issue persists even after latest AdGuard for Windows nightly update.

**Comment by danidoedel** [comment]
Seems to be resolved with the latest nightly (7.0.2430.6135).
